fixtures/customize/dest.bu

```diff
@@ -23,7 +23,7 @@ systemd:
         [Service]
         Type=oneshot
         RemainAfterExit=true
-        ExecStart=/bin/grep -qz 'serial --unit=0 --speed=115200 --word=8 --parity=no.terminal_input serial.terminal_output serial.#' /boot/grub2/grub.cfg
+        ExecStart=/bin/grep -qz 'serial --unit=0 --speed=115200 --word=8 --parity=no.terminal_input serial.terminal_output serial.#' /boot/grub2/grub.cfg /boot/grub2/console.cfg
```

Comment:
Hmm, I know I'm the one who suggested this, but actually I think we can just remove `/boot/grub2/grub.cfg` here. I initially kept it so that it would also work on media created without osbuild, but it won't because `console.cfg` will not exist and the command will fail trying to open it.

So either we make this a bash call instead that e.g. does `if ...` or that passes a globby `/boot/grub2/*.cfg`, or we just cut over to `console.cfg` only. I think let's just cut over. I don't foresee us rolling back the osbuild-by-default flip in cosa, but if we do we'll just have to revert this commit or enhance it.

Reply:
ah ok I have switched to just console.cfg and updated the commit. Let me test locally.